FlowControllerKit/UIStoryboard-Instantiate.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+extension UIStoryboard {
+
+    /// This is a convenience method that calls the method instantiateViewController(withIdentifier:)
+    /// and derives its identifier from the class name and returns the correct type so no type casting
+    /// is nececery.
+    ///
+    /// The storyboard ID needs to be set to the class name for this to works if not this will throw an exeption.
+    ///
+    /// - Parameter type: The ViewController type ypu want to inistantiate
+    /// - Returns: An instantiated instance of the ViewController
```

Comment:
I'm not sure about this but maybe block comments would be preferred here?

Reply:
There are multiple places in case the statement above is correct

Reply:
This is the preferred way to write documentation for Swift and also this is the way xcode auto generates documentation.